### ISSUE TITLE
Combining syn.set_spike_times and syn.set_spike_times_w_netstim not possible

### ISSUE THREAD

**Issue by espenhgn** [LFPy/LFPy#134]
Turns of using the two different methods to set spike times at the same time is not possible. 
```
cell = LFPy.Cell(morphology=os.path.join(LFPy.__path__[0], 'test',
                                         'ball_and_sticks.hoc'),
                passive=True)
t0 = 10.
t1 = 30.
tau = 5.
syn0 = LFPy.Synapse(cell=cell, idx=0, syntype='ExpSynI',
                   weight=1., tau=tau, record_current=True,
                   record_potential=True)
syn0.set_spike_times_w_netstim(noise=0., start=t0-1, number=1) # -1 because conduction delay

syn1 = LFPy.Synapse(cell=cell, idx=0, syntype='ExpSynI',
                   weight=1., tau=tau, record_current=True,
                   record_potential=True)
syn1.set_spike_times(np.array([t1]))

cell.simulate()

i0 = np.zeros(cell.tvec.size)
i0[cell.tvec > t0] = -np.exp(-np.arange((cell.tvec > t0).sum())*cell.dt / tau)

i1 = np.zeros(cell.tvec.size)
i1[cell.tvec > t1] = -np.exp(-np.arange((cell.tvec > t1).sum())*cell.dt / tau)

fig, axes = plt.subplots(3, 1, figsize=(10, 10))
axes[0].plot(cell.tvec, i0)
axes[0].plot(cell.tvec, syn0.i)
axes[1].plot(cell.tvec, i1)
axes[1].plot(cell.tvec, syn1.i)
axes[2].plot(cell.tvec, cell.somav)
axes[2].plot(cell.tvec, syn0.v)
axes[2].plot(cell.tvec, syn1.v)
```
results in only activation from the netstim version, and one postsynaptic potential:
![image](https://user-images.githubusercontent.com/2492641/56897846-d9e45c00-6a8f-11e9-9287-6b4f2dc27df3.png)

**Comment by espenhgn** [comment]
Started working on a fix here: https://github.com/espenhgn/LFPy/tree/fix_137

**Comment by espenhgn** [comment]
#135 should now fix this issue. @torbjone, fine with merging?

**Comment by espenhgn** [comment]
Merged #135 , closing.
